graylog2-rest-models/src/main/java/org/graylog2/rest/models/alarmcallbacks/AlarmCallbackError.java

```diff
@@ -32,6 +33,6 @@
 
     @JsonCreator
     public static AlarmCallbackError create(@JsonProperty("error") String error) {
-        return new AutoValue_AlarmCallbackError(error);
+        return new AutoValue_AlarmCallbackError(Strings.nullToEmpty(error));
```

Comment:
This caused an exception when rendering the alarm callback history item, because error is not nullable. Since we didn't want to chase all the usages of this field in the web interface code, we decided to make it "" instead.